cmd/entrypoint-wrapper/main.go

```diff
@@ -72,13 +72,15 @@ type options struct {
 	waitTimeout    time.Duration
 	cmd            []string
 	client         coreclientset.SecretInterface
+	roKubeconfig   bool
 }
 
 func bindOptions(flag *flag.FlagSet) *options {
 	opt := &options{}
 	flag.BoolVar(&opt.dry, "dry-run", false, "Print the secret instead of creating it")
 	flag.StringVar(&opt.waitPath, "wait-for-file", "", "Wait for a file to appear at this path before starting the program")
 	flag.StringVar(&opt.waitTimeoutStr, "wait-timeout", "", "Used with --wait-for-file, maximum wait time before starting the program")
+	flag.BoolVar(&opt.roKubeconfig, "ro-kubeconfig", false, "If set do not copy kubeconfig to a writable location")
```

Comment:
Can we have a better name for the flag? Like `--read-only-kubeconfig`

Reply:
+1

Reply:
Sure